src/UE4SSProgram.cpp

```diff
@@ -1074,6 +1079,15 @@ namespace RC
         setup_mods();
         start_lua_mods();
         start_cpp_mods();
+        
+        if (Unreal::UnrealInitializer::StaticStorage::bIsInitialized) {
+            fire_unreal_init_for_cpp_mods();
+        }
+
+        if (is_program_started()) {
```

Comment:
```suggestion
        if (is_program_started())
        {
```
Code-style consistency.